app/views/admin/schools/cohorts/appropriate_bodies/edit.html.erb

```diff
@@ -0,0 +1,41 @@
+<% @question = "Who is the appropriate body for their #{school_cohort.start_year} programme?" %>
+<% content_for :title, @question %>
+
+<% content_for :before_content, govuk_back_link(text: "Back", href: admin_school_cohorts_path(school)) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_for @form, url: admin_school_cohort_appropriate_body_path, method: :patch do |form| %>
+      <%=
+        form.govuk_radio_buttons_fieldset(
+          :appropriate_body_id,
+          legend: { text: @question, tag: "h1", size: "l" },
+          hint: { text: "This will change the default for any new ECTs registered, but will not affect any existing ECTs" },
+          caption: { text: school.name }
+        ) do
+      %>
```

Comment:
Minor one, but I think we should move this caption out of the h1 here.  The captions can either be in or out of the h1, which visually appears the same, but for screen reader users it’s better here if the caption it outside, I think, as the school name doesn’t naturally form part of the question.

```suggestion
      <span class="govuk-caption-l"><%= school.name %></span>
      <%=
        form.govuk_radio_buttons_fieldset(
          :appropriate_body_id,
          legend: { text: @question, tag: "h1", size: "l" },
          hint: { text: "This will change the default for any new ECTs registered, but will not affect any existing ECTs" }
        ) do
      %>
```

Reply:
Updated